common/config/azure-pipelines/templates/integration-test-steps.yaml

```diff
@@ -173,3 +180,10 @@ steps:
       testResultsFiles: core/electron/lib/test/*_junit_results.xml
       testRunTitle: "Core - Electron Tests - ${{ parameters.nodeVersion }}"
     condition: succeededOrFailed()
+
+      - task: PublishTestResults@2
+    displayName: "Publish Frontend Tiles Test Results"
+    inputs:
+      testResultsFiles: extensions/frontend-tiles/lib/test/*_junit_results.xml
```

Comment:
what with the `*`? do we not know the file path?

Reply:
timestamps?

Reply:
fixed to now point to the specific file.